### PR TITLE
OPENSTACK-2682: add external network mapping for bigip-onboard

### PR DESCRIPTION
**Pull request by zhang-shengping**

### CODE REVIEW

**Comment by zhang-shengping** [comment]
image

File: f5_openstack_agent/client/bigip-onboard-cli.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ required params:
 - icontrol_hostname: BIG-IP icontrol API hostname
 - icontrol_username: BIG-IP icontrol API username
 - icontrol_password: BIG-IP icontrol API password
+- external-physical-mappings: maps of neutorn physical network to bigip interface or a trunk port. Default value is default:1.1.
```
Comment (zh805) [inline f5_openstack_agent/client/bigip-onboard-cli.md]: external-physical-mappings is a positional parama

File: f5_openstack_agent/client/bigip-onboard-cli.md [hunk 2]
```diff
@@ -48,7 +52,7 @@ required params:
 - id: ID of BIG-IP group
 
 positional params:
```
Comment (zh805) [inline f5_openstack_agent/client/bigip-onboard-cli.md:54]: add external-physical-mapppings to positional params

Comment (zhang-shengping) [reply f5_openstack_agent/client/bigip-onboard-cli.md:54]: yes, code has updated

**Comment by zhang-shengping** [comment]
https://gitswarm.f5net.com/openstack/f5-openstack-lbaasv2-agent-lite-pipeline/-/pipelines/6108907

**Review by zh805 — APPROVED** [review]

**Pull request merged**